app/views/webauthn_verifications/success.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @title = "Verify with Security Device" %>
+
+<div class="t-body">
```

Comment:
NIT: This could be a `<main>` element

Reply:
I'm not sure that'd be consistent with other templates, I don't see `<section>` in a code search.

Reply:
Hm, fair point Jacques! I guess we can open the can of worms around consistency vs. correctness. I favour using semantic elements over `div`s where appropriate. I'd argue this is more semantically correct and the other templates could benefit from be updated. 

That said, I'm not too fussed on _this_ template in particular having a section or not since it's ultimately only one section. If there were multiple sections, I'd have a stronger opinion 😅 .

Reply:
It would be a great spring cleaning project to convert to `<section>`s!

Reply:
^^^ could be a nice project to do a migration over to some semantic elements and create/enforce a style guide for this kind of stuff ([ref](https://github.com/rubygems/rubygems.org/pull/2997#issuecomment-1079084829)).